content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -1190,22 +1189,21 @@ Properties are **key-value storages**. They can be set within a chat, a thread,
 ```json
 {
   "id": "K600PKZON8",
-  "timestamp": 1473433500,
   "active": true,
   "user_ids": ["smith@example.com"],
   "restricted_access": true,
   "events": [
     // array of "Event" objects
   ],
-  "order": 112057129857,
   "properties": {
     // "Properties" object
   },
   "access": {
     // "Access" object
   },
   "previous_thread_id": "K600PKZOM8",
-  "next_thread_id": "K600PKZOO8"
+  "next_thread_id": "K600PKZOO8",
+  "created_at": "2020-05-07T07:11:28.288340Z"
```

Comment:
Could also mention this param in the table? Something like:

created_at | Date & time format with a resolution of microseconds, `UTC string`. Generated server-side.